docs/resources/gaussdb_opengauss_instance.md

```diff
@@ -336,19 +346,19 @@ $ terraform import huaweicloud_gaussdb_opengauss_instance.test <id>
 
 Note that the imported state may not be identical to your resource definition, due to the attribute missing from the
 API response. The missing attributes include: `password`, `ha.0.mode`, `ha.0.instance_mode`, `configuration_id`,
-`disk_encryption_id`, `enable_force_switch`, `enable_single_float_ip`, `parameters`, `period_unit`, `period` and
-`auto_renew`. It is generally recommended running `terraform plan` after importing a GaussDB OpenGauss instance. You can
-then decide if changes should be applied to the GaussDB OpenGauss instance, or the resource definition should be updated
-to align with the GaussDB OpenGauss instance. Also you can ignore changes as below.
+`disk_encryption_id`, `enable_force_switch`, `enable_single_float_ip`, `parameters`, `advance_features`, `period_unit`,
+`period` and `auto_renew`. It is generally recommended running `terraform plan` after importing a GaussDB OpenGauss
+instance. You can then decide if changes should be applied to the GaussDB OpenGauss instance, or the resource definition
+should be updated to align with the GaussDB OpenGauss instance. Also you can ignore changes as below.
 
 ```hcl
 resource "huaweicloud_gaussdb_opengauss_instance" "test" {
   ...
 
   lifecycle {
     ignore_changes = [
-      password, configuration_id, disk_encryption_id, enable_force_switch, enable_single_float_ip, parameters, period_unit,
-      period, auto_renew,
+      password, configuration_id, disk_encryption_id, enable_force_switch, enable_single_float_ip, parameters,
+      advance_features, period_unit, period, auto_renew,
```

Comment:
advance_features is saved in the read func, it should not need to ignore it here